rush.json

```diff
@@ -2241,6 +2241,11 @@
       "packageName": "@azure-rest/ai-inference",
       "projectFolder": "sdk/ai/ai-inference-rest",
       "versionPolicyName": "client"
+    },
+    {
```

Comment:
We have changes in rush.json file because this is newly RP to release. But our merge is blocked by this rule https://github.com/Azure/azure-sdk-for-js/blob/main/.github/CODEOWNERS#L1543

![image](https://github.com/user-attachments/assets/899b8807-e7d2-44b2-a185-ababf5941ea2)